sklearn/metrics/_ranking.py

```diff
@@ -622,6 +623,9 @@ def _multiclass_roc_auc_score(
         ``'weighted'``:
             Calculate metrics for each label, taking into account the
             prevalence of the classes.
+        ``'micro'``:
```

Comment:
`'micro'` needs to be appended to the set of valid values for `average` at l.616.

Reply:
catch, thanks!